common/ta_errors.h

```diff
@@ -180,35 +175,36 @@ typedef enum {
   /**< Failed to open file */
 
   // UTILS module
+  SC_UTILS_NULL = 0x01 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
   /**< NULL object in utils */
-  SC_UTILS_WRONG_REQUEST_OBJ = 0x01 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
-  /**< Wrong TA request object */
-  SC_UTILS_TIMER_ERROR = 0x02 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
+  SC_UTILS_OOM = 0x02 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
```

Comment:
Why we need to independent OOM status code for each module?

Reply:
Let me integrate